data/json/itemgroups/Clothing_Gear/practical_survivor_items.json

```diff
@@ -0,0 +1,162 @@
+[
+  {
+    "id": "practical_pants",
+    "type": "item_group",
+    "subtype": "distribution",
+    "//": "Assortment of practical pants characters would wear during the apocalpyse.",
+    "items": [ { "item": "pants_army" }, { "item": "pants_cargo" }, { "item": "jeans" }, { "item": "motorbike_pants" } ]
+  },
+  {
+    "id": "practical_skintight_tops",
+    "type": "item_group",
+    "subtype": "distribution",
+    "//": "Assortment of practical under shirts and underwear characters would wear during the apocalpyse.",
+    "items": [
+      { "item": "army_top" },
+      { "item": "tank_top" },
+      { "item": "under_armor" },
+      { "item": "tshirt" },
+      { "item": "tshirt_tour" },
+      { "item": "longshirt" }
+    ]
+  },
+  {
+    "id": "practical_regular_tops",
+    "type": "item_group",
+    "subtype": "distribution",
+    "//": "Assortment of practical normal layer clothes characters would wear during the apocalpyse.",
+    "items": [
+      { "item": "combat_shirt" },
+      { "item": "hoodie" },
+      { "item": "sheriffshirt" },
+      { "item": "sweater" },
+      { "item": "sweatshirt" }
+    ]
+  },
+  {
+    "id": "practical_bottoms_underwear",
+    "type": "item_group",
+    "subtype": "distribution",
+    "//": "Assortment of practical underwear characters would wear during the apocalpyse.",
+    "items": [
+      { "item": "boxer_briefs" },
+      { "item": "boxer_shorts" },
+      { "item": "leggings" },
+      { "item": "boy_shorts" },
+      { "item": "briefs" },
+      { "item": "panties" },
+      { "item": "under_armor_shorts" }
```

Comment:
```suggestion
      { "item": "under_armor_shorts" },
      { "item": "shorts" }
```

Reply:
shorts are normal layer IIRC

Reply:
Oh, sorry, I'm doing this on mobile so I remembered wrong.

Reply:
all good I appreciate the suggestions :)